internal/credit/credit.go

```diff
@@ -227,12 +227,11 @@ func (e *LockErrNotObtainedError) Error() string {
 
 type LedgerAlreadyExistsError struct {
 	Namespace string
-	LedgerID  ulid.ULID
-	Subject   string
+	Ledger    Ledger
 }
 
 func (e *LedgerAlreadyExistsError) Error() string {
-	return fmt.Sprintf("ledger (%s.%s) already exitst for subject %s", e.Namespace, e.LedgerID, e.Subject)
+	return fmt.Sprintf("ledger (%s.%s) already exitst for subject %s", e.Namespace, e.Ledger.ID, e.Ledger.Subject)
```

Comment:
Should we omit the namespace from the error? It's largely transparent anyway. It might be more confusing than useful.